docs/interactions/Message_Components.md

```diff
@@ -215,10 +215,22 @@ There are 5 different [select menu components](#DOCS_INTERACTIONS_MESSAGE_COMPON
 
 The string select menu (type `3`) is the *only* select type that allows (and *requires*) apps to define the `options` that appear in the dropdown list. The other 4 select menu components (users, roles, mentionables, and channels) are auto-populated with options corresponding to the resource type—similar to [command option types](#DOCS_INTERACTIONS_APPLICATION_COMMANDS/application-command-object-application-command-option-type).
 
-In addition to the `values` array in all [select menu interaction payloads](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-menu-interaction), auto-populated select menu components (users, roles, mentionables, and channels) also include an additional [`resolved` object](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-menu-resolved-object) that provides additional details about the user's selected resource.
+The interaction response for auto-populated select menu components also includes a [`resolved` object](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-menu-resolved-object) that provides additional details about the user's selected resource.
 
 The payloads for the select menu components are detailed in the [select menu structure](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-menu-structure).
 
+### Select Menu Interaction Values
+
+[Select menu interaction payloads](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/select-menu-object-select-menu-interaction) include a `values` array,
+which contains any values selected by the user. The underlying type of this array varies depending on the type of select menu used.
+
+| Select Menu Type                                         | `values` Array Type | Description                                          |
+| -------------------------------------------------------- | ------------------- | ---------------------------------------------------- |
+| String (`3`)                                             | strings             | The `value` field from any of the selected `options` |
+| User (`5`), Role (`6`), Mentionable (`7`), Channel (`8`) | snowflakes          | The IDs of any of the selected resources             |
+
+- String select menus (type `3`) will
+
```

Comment:
```suggestion
```

I don't think this information really belongs in a table or in this format. I think to properly address it, we'd need to revisit how we talk about message components in general and how information is split up between "Receiving and Responding" and the "Message Components" pages.

However, that's out of the scope of this PR for sure, and so I'd just put the information in `Receiving_and_Responding.md` instead (i made a suggestion to the table and table note accordingly).